.github/workflows/build-deb-ubuntu-22.04.yml

```diff
@@ -0,0 +1,190 @@
+name: Build Release debian package - Ubuntu 22.04
+run-name: Build Release debian package - Ubuntu 22.04
+on:
+  workflow_dispatch:
+  workflow_call:
+    inputs:
+      ref:
+        default: ${{ github.ref }}
+        required: false
+        type: string
+      MAJOR:
```

Comment:
If I'm understanding correctly, we need to provide these when we trigger a release?

Reply:
Those get set from https://github.com/warthog9/luwen/blob/ce60d97d80dbc169cd1c8aeb790a901dc9125222/.github/workflows/release.yml#L100 (assuming that link continues to work), basically release-kit/semver (https://github.com/release-kit/semver) sorts what they are

and

https://github.com/warthog9/luwen/blob/ce60d97d80dbc169cd1c8aeb790a901dc9125222/.github/workflows/release.yml#L59

starts the passing around and

things like https://github.com/warthog9/luwen/blob/ce60d97d80dbc169cd1c8aeb790a901dc9125222/.github/workflows/release.yml#L232

set things up